libs/blog/articles/data-access/src/lib/guards/article-exists.guard.ts

```diff
@@ -45,7 +45,7 @@ export const articleExistsGuard: CanActivateFn = (route) => {
               {},
             );
           } else {
-            return notFoundPageUrlTree;
+            return router.createUrlTree(['/pl']);
```

Comment:
should we include here also articles with missing english translations and redirect to the '/' then?
And what will happen if you redirect to the '/' and you're currently on the polish version?